visitors/README.md

```diff
@@ -0,0 +1,24 @@
+# Visitor Prototype
+
+Synopsis:
+
+    utk BIOS OPERATIONS...
+
+Examples:
+
+    # Remove two files by their GUID and replace
+    utk winterfell.rom \
+      remove 12345678-9abc-def0-1234-567890abcdef \
+      remove 00000000-0000-0000-0000-000000000000 \
+      replace_pe32 Shell linux.efi \
+      save winterfell2.rom
+
+Operations:
+
+- `remove (GUID|NAME)`: Remove the first file which matches the given GUID or
+  NAME. If the number of matches is unequal to one, an error results.
+- `replace (GUILD|NAME) FILE`: Replace the first file which matches the given
+  GUID or NAME with the contents of FILE. If the number of matches is unequal
+  to one, an error results.
+- `assemble FILE`: Re-assemble the BIOS and save to the given file.
```

Comment:
assemble should have some directory to assemble from? also it doesn't make sense to pass utk a bios image if you already want to assemble it from the working directory

Reply:
I renamed this to `save`. Read the new README and see if that makes sense.